libbeat/cfgfile/reload.go

```diff
@@ -15,3 +31,158 @@ type ReloadConfig struct {
 	Period  time.Duration `config:"period"`
 	Enabled bool          `config:"enabled"`
 }
+
+type Wrapper interface {
```

Comment:
I like `RunnerFactory` and `Create`. For the `ID()` agree, but still need to do some cleanup to get there ...